pkg/client/cache/index.go

```diff
@@ -24,7 +24,11 @@ func ImageStreamReferenceIndexFunc(obj interface{}) ([]string, error) {
 			}
 			params := trigger.ImageChangeParams
 			name, _, _ := imageapi.SplitImageStreamTag(params.From.Name)
-			keys = append(keys, params.From.Namespace+"/"+name)
+			namespace := params.From.Namespace
```

Comment:
anything containing this code will be reading using a client library that does the defaulting, right?